### PR TITLE
Improvement of DM UT from 15% to 20%

### PR DESCRIPTION
**Pull request by vineela-pachchipulusu**
The total statements covered  for the DM code is increased from 15-20%

Here is the coverage report
[coverage.txt](https://github.com/Wind-River/cloud-platform-deployment-manager/files/14551440/coverage.txt)

### CODE REVIEW

File: controllers/common/common_test.go [hunk 1]
```diff
@@ -68,6 +116,167 @@ var _ = Describe("Common utils", func() {
...
+				SyncIFNameByUuid(profile, current)
+				Expect(profile.Interfaces.Ethernet[0].Port.Name).To(Equal(profPortName))
+				Expect(profile.Interfaces.Ethernet[0].CommonInterfaceInfo.Name).To(Equal(profEthName))
+
+			})
+		})
+	})
+	Describe("Test processLines", func() {
+		Context("When there given non empty lines", func() {
+			It("SHould gather delta config", func() {
```
Comment (sselvara1) [inline controllers/common/common_test.go:252]: Should - case sensitive

**Comment by sselvara1** [comment]
> The total statements covered for the DM code is increased from 15-20%
> 
> Here is the coverage report [coverage.txt](https://github.com/Wind-River/cloud-platform-deployment-manager/files/14551440/coverage.txt)

Commit message can be updated as below: 

Increment DM UT coverage to 20%

The overall UT coverage of DM repo is incremented 
from 15% to 20%. New test files are added for few
of the webhooks.

Signed-off-by: vineela-pachchipulusu <Vineela.Pachchipulusu@windriver.com>

**Comment by vineela-pachchipulusu** [comment]
> > The total statements covered for the DM code is increased from 15-20%
> > Here is the coverage report [coverage.txt](https://github.com/Wind-River/cloud-platform-deployment-manager/files/14551440/coverage.txt)
> 
> Commit message can be updated as below:
> 
> Increment DM UT coverage to 20%
> 
> The overall UT coverage of DM repo is incremented from 15% to 20%. New test files are added for few of the webhooks.
> 
> Signed-off-by: vineela-pachchipulusu [Vineela.Pachchipulusu@windriver.com](mailto:Vineela.Pachchipulusu@windriver.com)

DOne

File: api/v1/hostprofile_type_test.go [hunk 2]
```diff
@@ -19,6 +20,187 @@ var _ = Describe("HostProfile controller", func() {
...
+
+	Context("Test StringsToPtpInterfaceItemList func", func() {
+		It("Should return the array of PtpInterfaceItem with input items", func() {
+			items := []string{"item1", "item2"}
+			item1 := PtpInterfaceItem("item1")
+			item2 := PtpInterfaceItem("item2")
+			list := make([]PtpInterfaceItem, 0)
+			list = append(list, item1, item2)
+			exp := PtpInterfaceItemList(list)
+			got := StringsToPtpInterfaceItemList(items)
```
[the remark this answers is not in the document]
Comment (vineela-pachchipulusu) [reply api/v1/hostprofile_type_test.go:134]: Yes it calls PtpInterfaceItemList func internally

[the remark this answers is not in the document]
Comment (sselvara1) [reply api/v1/hostprofile_type_test.go:134]: hmm ok. So, we are ensuring StringsToPtpInterfaceItemList is covered.

**Review by sselvara1 — APPROVED** [review]

**Pull request merged**